appveyor.yml

```diff
@@ -90,6 +75,10 @@ build_script:
 
     $datatable_wheel = ls dist\*-cp36-*.whl
 
+    echo "----- build_info 3.6 ---------------------------------------------"
+    cat src/datatable/_build_info.py
+    echo "------------------------------------------------------------------"

```

Comment:
Also, here, you might need to add new lines between every command, otherwise they will all stick together and produce an error.

Reply:
you're right, they did end up stuck together:
```
# Build and test wheel for Python 3.6
# =======================================================================
$env:PATH = "C:\Python36-x64;C:\Python36-x64\Scripts;$DEFAULT_PATH"
python ci\ext.py wheel
$datatable_wheel = ls dist\*-cp36-*.whl
echo "----- build_info 3.6 ---------------------------------------------" cat src/datatable/_build_info.py echo "------------------------------------------------------------------"
pip --disable-pip-version-check install $datatable_wheel pytest docutils pandas
pytest .\tests\
```

Reply:
Yeah, this only happens with multiline commands for some reason...